### PR TITLE
Change/add various Google Services

### PR DESCRIPTION
**Pull request by Kefaku**
* added the following Google Services similar to `Blogger`, `Gmail` and `YouTube`: `Google One` and `Google Pay`
* added reference to `Google` to the following Google Services: `Blogger`, `Gmail`, `YouTube`, `Google One` and `Google Pay`
* changed `YouTube` difficulty to `medium`, because it requires a confirmation email
* added domains `google.co.uk` and `takeout.google.com` to `Google`

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -1691,7 +1691,7 @@
         "name": "Blogger",
         "url": "https://support.google.com/blogger/answer/41932",
         "difficulty": "impossible",
-        "notes": "You can't delete your Blogger Account without deleting your entire Google Account. But you can delete your blog.",
+        "notes": "You can't delete your Blogger Account without deleting your entire Google Account. But you can delete your blog.\nIf you want to delete your entire google account, use the instructions for <a  href=\"https://justdeleteme.xyz/#google\" target=\"_self\">Google</a>.",
```
Comment (tupaschoal) [inline _data/sites.json:1694]: Why the `target=self`, so it opens on the same page?

Comment (Kefaku) [reply _data/sites.json:1694]: Yeah, not really necessary, but that's what it does

Comment (Kefaku) [reply _data/sites.json]: done @tupaschoal, anything else I should change?

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**